src/routes.ts

```diff
@@ -1,7 +1,7 @@
 import {Router} from 'lib/routes/router'
 
 export const router = new Router({
-  Home: '/',
+  Home: ['/', '/start/:name/:rkey'],
```

Comment:
If you give an array to this, it will match multiple URL paths to the same React route. So in this case both `https://bsky.app/` and `https://bsky.app/profile/start/:name/:rkey` will route to `Home`. That we handle here

https://github.com/bluesky-social/social-app/blob/077da0830924d4c040ef095d8349f13fdfdf4372/src/lib/routes/router.ts#L3

Reply:
Nice thanks